dozer-types/src/types/field.rs

```diff
@@ -261,9 +262,9 @@ impl Field {
         }
     }
 
-    pub fn as_bson(&self) -> Option<&[u8]> {
+    pub fn as_json(&self) -> Option<JsonValue> {
```

Comment:
I think `as_*` methods are not supposed to do cloning, let's return `Option<&JsonValue>`

Reply:
yup!